admin/lib/generators/solidus_admin/install/install_generator.rb

```diff
@@ -10,6 +10,14 @@ def install_solidus_core_support
           }
         RUBY
       end
+
+      def ignore_tailwind_build_files
+        append_file(".gitignore", "app/assets/stylesheets/solidus_admin/builds/") if File.exist?(Rails.root.join(".gitignore"))
+      end
+
+      def build_tailwind
+        rake "solidus_admin:tailwindcss:build"
```

Comment:
Why is this task needed during the installation?

Reply:
Because the compiled CSS is generated in the host application, that's to allow users to extend the admin style, for instance, using other tailwind classes or stylesheets.

Reply:
Got it! I somehow thought that was already part of the watch task. Do you think it might be a good idea to always run it when the watch task is executed?

Reply:
@waiting-for-dev would it still be possible to ship a compiled CSS that won't require tailwind to be installed for all those not needing to customize?

Reply:
I gave more thought to it, and I think it'd be possible, but at the expense of a more complex setup.

We could keep the generation of the CSS for the solidus_admin engine within the development process of it. Then, let's consider a host application that needs to reuse. It should install tailwind, and we could have a different task using the same tailwind config file (the erb template) from the engine but generating the CSS output file to an application path matching the one in solidus_admin.

If we add other engines into the mix, each one should do something similar to what I described for the host application. However, each of them should ship an isolated CSS file, so we should change the solidus_admin manifest to take all of them.

I wonder if it pays off, as now customization is more streamlined for both scenarios. Happy to explore it further if you think that would be a good idea, but I'd prefer to do that on another PR. We can keep it in the back of our minds as a future optimization if we feel the need for it.